interfaces/builtin/gsettings_test.go

```diff
@@ -83,8 +90,10 @@ func (s *GsettingsInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	snippet, err := s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecurityAppArmor)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
-	// connected plugs have a non-nil security snippet for seccomp
```

Comment:
Hrm, this change is confusing - but it looks like the original test was simply misguided. Oh well. But this is fine :)